recipes/docker_cgroup.rb

```diff
@@ -39,9 +39,9 @@
     user 'root'
     group 'root'
     code <<-EOH
-        echo -e "#{docker_cgroup_cpu_cfs_quota_us} #{cpu_quota_period}" > #{node['hops']['cgroup']['mount-path']}/docker.slice/cpu.max
-        echo #{docker_hard_limit_memory_bytes + excess_memory} > #{node['hops']['cgroup']['mount-path']}/docker.slice/memory.max
-        echo #{docker_soft_limit_memory_bytes + excess_memory} > #{node['hops']['cgroup']['mount-path']}/docker.slice/memory.high
+        echo -e "#{docker_cgroup_cpu_cfs_quota_us} #{cpu_quota_period}" > #{node['hops']['cgroup']['mount-path']}/#{node['hops']['docker']['cgroup']['directory']}/cpu.max
```

Comment:
The attribute is wrong. Shouldn't it be `node['hops']['docker']['cgroup']['parent']` ?